packaging/os/yumrepo.py

```diff
@@ -0,0 +1,501 @@
+#!/usr/bin/python
+# encoding: utf-8
+
+# (c) 2015, Jiri Tyr <jiri.tyr@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from ansible.constants import get_config
+from ansible.constants import load_config_file
```

Comment:
I don't think you can use those in modules.

The way Ansible modules work: your module's source code is pre-processed, treating `from ansible.module_utils.SOMETHING import *` as an include directive, then some placeholders in those includes get replaced with module arguments etc., then the preprocessed source is SCPed over to the remote and executed as a single file.

There's no requirement for Ansible to be installed on remote hosts, so you can't rely on ansible imports succeeding, with the exception of those special include directives that masquerade as Python imports.